packages/calling/src/CallHistory/types.ts

```diff
@@ -41,4 +50,16 @@ export interface ICallHistory extends Eventing<CallHistoryEventTypes> {
     sort: SORT,
     sortBy: SORT_BY
   ): Promise<JanusResponseEvent>;
+  /**
+   * This API `updateMissedCalls` is utilized to Updates the read state of user missed calls based on the specified parameters.
```

Comment:
It should be update the read state not updates

Reply:
Done